### DISCUSSION TITLE
How to infer the shadow of cars?

### DISCUSSION THREAD

**Discussion by CSU-NXY** [OPEN-AIR-SUN/mars#13] [General]
Hi, thanks for sharing your great work.

I noticed that the shadow looks realistic even when you move objects. I'm quite curious about it. Did you train the shadow with cars, or use an independent shadow model?

**Comment by JiantengChen** [comment]
Hi! It's great to receive your feedback! We didn‘t use the independent shadow model, our shadow is decoupled from the background along with the cars.

**Comment by wuzirui** [comment]
Actually, we think that it is the shared category-level object representation that helps the model to decompose shadow and ground plane appearances. The editing performance when we use instance-wise object representation indeed drops in shadow regions.

**Comment by CSU-NXY** [comment]
Thanks!
